build.gradle

```diff
@@ -4258,6 +4258,8 @@ task javadoc(type: Javadoc, dependsOn: createMSPfile) {
 
     options.addStringOption("-since").setValue((9..jfxReleaseMajorVersion.toInteger()).join(","))
     options.addStringOption("-since-label").setValue("New API since JavaFX 9")
+    options.addStringOption("Xmaxwarns").setValue("1000")
+    options.addStringOption("Xmaxerrs").setValue("1000")
```

Comment:
would it make sense to treat warnings as errors?  at least once we fix JBS-TBD to fix all the warnings?

Reply:
Eventually, yes. Which is what I noted in the JBS issue.

Btw, I filed [JDK-8314590](https://bugs.openjdk.org/browse/JDK-8314590) to track fixing all the warnings. A follow-up to _that_ issue will be to enable treating warnings as errors.

Reply:
Thank you for clarification, Kevin!
I wish github would copy bug description from the ticket to the PR... I ddi not bother to read JBS this time.

Reply:
> I wish github would copy bug description from the ticket to the PR... I ddi not bother to read JBS this time.

I'm glad it doesn't given what ends up in the Descriptions for many bugs. I should have added that to the PR description.